### PR TITLE
Fixes #6153 - explain parameter help comments

### LINKED ISSUE #6153: Explain how to use comments in parameter for help
Follow up to https://github.com/MicrosoftDocs/PowerShell-Docs/pull/6140

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary
Fixes [AB#1738357](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1738357) - Fixes #6153 - explain parameter help comments

## PR Context

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [x] Version 7.1 preview content
- [x] Version 7.0 content
- [x] Version 6 content
- [x] Version 5.1 content

**Conceptual content**
- [ ] Fundamental conceptual articles
- [ ] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles
- [ ] Community content

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Comment_Based_Help.md [hunk 1]
```diff
@@ -174,9 +174,9 @@ In a script module `.psm1`, comment-based help uses the syntax for functions,
 not the syntax for scripts. You cannot use the script syntax to provide help
 for all functions defined in a script module.
 
-For example, if you are using the "ExternalHelp" keyword to identify the
+For example, if you are using the `.ExternalHelp` keyword to identify the
 XML-based help files for the functions in a script module, you must add an
-"ExternalHelp" comment to each function.
+`.ExternalHelp comment to each function.
```
Comment (chasewilson) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Comment_Based_Help.md:179]:
```suggestion
`.ExternalHelp` comment to each function.
```

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Comment_Based_Help.md [hunk 2]
```diff
@@ -223,20 +223,41 @@ function or script syntax determines the order in which the parameters (and
...
 
+You can also specify a parameter description by placing a comment in the
+function or script syntax immediately before the parameter variable name. For
+this to work, you must also have a comment block with at least one keyword.
+
+If you use both a syntax comment and a `.PARAMETER` keyword, the description
+associated with the `.PARAMETER` keyword is used, and the syntax comment is
+ignored.
+
+```
```
Comment (chasewilson) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Comment_Based_Help.md:234]: Should this be a PowerShell code fence?

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Comment_Based_Help.md [hunk 3]
```diff
@@ -286,10 +307,11 @@ provider.
 
 ### .FORWARDHELPCATEGORY
 
-Specifies the help category of the item in "ForwardHelpTargetName". Valid
-values are "Alias", "Cmdlet", "HelpFile", "Function", "Provider", "General",
-"FAQ", "Glossary", "ScriptCommand", "ExternalScript", "Filter", or "All". Use
-this keyword to avoid conflicts when there are commands with the same name.
+Specifies the help category of the item in `.ForwardHelpTargetName`. Valid
+values are **Alias**, **Cmdlet**, **HelpFile**, **Function**, **Provider**,
```
Comment (chasewilson) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Comment_Based_Help.md:311]: per our other conversation about valid values, these should be code fenced

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Comment_Based_Help.md (hunk 2 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Core/About/about_Comment_Based_Help.md:234]: Fixed

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Comment_Based_Help.md (hunk 3 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Core/About/about_Comment_Based_Help.md:311]: Fixed

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Comment_Based_Help.md (hunk 1 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Core/About/about_Comment_Based_Help.md:179]: Fixed

**Pull request merged**